openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -122,6 +122,10 @@ def is_course_cohorted(course_key):
     return _get_course_cohort_settings(course_key).is_cohorted
 
 
+def get_course_cohort_id(course_key):
+    return _get_course_cohort_settings(course_key).id
```

Comment:
Can you add documentation about what this method is? I know it's related to the Backbone view, but it would not be obvious to others.